docs/debugging.md

```diff
@@ -0,0 +1,90 @@
+# Debugging
+
+## Connecting to VMs Using SSH/RDP
+- To debug Linux VMs, SSH can be used to login to the machines for troubleshooting and viewing log files. 
+- To debug Windows VMs, a RDP client such as Windows Remote Desktop on Windows or xfreerdp on Linux can be used to login to the machines for troubleshooting and viewing log files.
+- Workstation VMs are not exposed to the internet and do not have public IPs, a bastion host such as the DC or Connector can be used to access the Workstation VMs on the private network.
+
+### Connecting to CentOS Workstations
+One way to access CentOS Workstations is to use the Connector as a bastion host. Execute the following command to first SSH into the Connector:
+
+```ssh -A <login-user>@<cac-public-ip>``` 
```

Comment:
add -i to indicate which ssh key to use. explain to use the private key corresponding to cac_admin_ssh_pub_key_file or centos_admin_ssh_pub_key_file for GCP or admin_ssh_pub_key_file for AWS.
BTW we should make it the same for AWS and GCP - use the same key for both cac and centos

Reply:
I've added this on my list of to do's.